Mathlib/Order/Hom/Basic.lean

```diff
@@ -924,6 +924,12 @@ theorem lt_iff_lt (e : α ≃o β) {x y : α} : e x < e y ↔ x < y :=
 
 @[gcongr] protected alias ⟨_, GCongr.orderIso_apply_lt_apply⟩ := lt_iff_lt
 
+theorem lt_symm_apply (e : α ≃o β) {x : α} {y : β} : x < e.symm y ↔ e x < y := by
+  rw [← e.lt_iff_lt, e.apply_symm_apply]
+
+theorem symm_apply_lt (e : α ≃o β) {x : α} {y : β} : e.symm y < x ↔ y < e x := by
+  rw [← e.lt_iff_lt, e.apply_symm_apply]
```

Comment:
If you're adding these, please add `le` versions too (if they don't exist yet). Otherwise LGTM. Thanks!
bors d+

Reply:
They already exist.

bors merge